docs/featurestore/integrations/guides/sagemaker.rst

```diff
@@ -1,91 +1,136 @@
+    :target: `sagemaker_secrets_manager.png`_
+    :scale: 40 %
     :align: center
-    :scale: 20 %
     :figclass: align-center
 
-As secret name enter *hopsworks/role/[MY_SAGEMAKER_ROLE]* replacing [MY_SAGEMAKER_ROLE] with the AWS role used by the SageMaker instance that should access the Feature Store. Select next twice and finally store the secret. Then click on the secret in the secrets list and take note of the *Secret ARN*.
+As secret name enter *hopsworks/role/[MY_SAGEMAKER_ROLE]* replacing [MY_SAGEMAKER_ROLE] with the AWS
+role used by the SageMaker instance that should access the Feature Store. Select next twice and finally
+store the secret. Then click on the secret in the secrets list and take note of the *Secret ARN*.
 
-.. _hopsworks_secrets_manager2.png: ../../../_images/secrets_manager2.png
-.. figure:: ../../../imgs/cloud/secrets_manager2.png
-    :alt: Hopsworks feature store secrets manager step 2
-    :target: `hopsworks_secrets_manager2.png`_
+.. _sagemaker_secrets_manager2.png: ../../../_images/sagemaker_secrets_manager2.png
+.. figure:: ../../../imgs/feature_store/sagemaker_secrets_manager2.png
+    :alt: Storing the Feature Store API Key in the Secrets Manager Step 2
+    :target: `sagemaker_secrets_manager2.png`_
     :align: center
-    :scale: 30 %
+    :scale: 35 %
     :figclass: align-center
 
-**(Alternative 2 step 2) Granting access to the secret to the SageMaker notebook role**
+**Granting access to the SecretsManager to the SageMaker notebook role**
 
-In the AWS management console go to *IAM*, select *Roles* and then the role that is used when creating SageMaker notebook instances. Select *Add inline policy*. Choose *Secrets Manager* as service, expand the *Read* access level and check *GetSecretValue*. Expand Resources and select *Add ARN*. Paste the ARN of the secret created in the previous step. Click on *Review*, give the policy a name und click on *Create policy*.
+In the AWS management console go to *IAM*, select *Roles* and then the role that is used when creating
+SageMaker notebook instances. Select *Add inline policy*. Choose *Secrets Manager* as service, expand the
+*Read* access level and check *GetSecretValue*. Expand Resources and select *Add ARN*. Paste the ARN of
+the secret created in the previous step. Click on *Review*, give the policy a name und click on *Create policy*.
 
-.. _hopsworks_aws_policy.png: ../../../_images/aws_policy.png
-.. figure:: ../../../imgs/cloud/aws_policy.png
-    :alt: Hopsworks feature store set policy
-    :target: `hopsworks_aws_policy.png`_
+.. _sagemaker_aws_policy.png: ../../../_images/sagemaker_aws_policy.png
+.. figure:: ../../../imgs/feature_store/sagemaker_aws_policy.png
+    :alt: Configuring the access policy for the Secrets Manager
+    :target: `sagemaker_aws_policy.png`_
     :align: center
-    :scale: 30 %
     :figclass: align-center
 
-**Installing hopsworks-cloud-sdk and connecting to the Feature Store**
+Step 3: Installing the Pandas SDK
```

Comment:
Pandas?

Reply:
I'm trying to get more clear naming in the Databricks guide and call them Pandas SDK and Spark DataFrames SDK for now. Maybe not a good idea but hops and hopsworks-cloud-sdk is just really confusing.